apis/sgv2-quarkus-common/src/main/resources/application.yaml

```diff
@@ -152,3 +152,15 @@ quarkus:
               datastore:
                 configuration:
                   DataStorePropertiesConfiguration/fetchSupportedFeatures/Retry/maxRetries: 0
+
+# shared production properties
+"%prod":
```

Comment:
I am actually not sure if I like the mixup of the production and development properties in one file.. I am used to have

* `application-dev.yaml`
* `application-prod.yaml` 

which clearly separates those.. The ``%dev` and `%prod` notation from quarkus is mixing everything in one file.. Should we change?

Reply:
I agree that it gets bit messy. Assuming `application.yaml` has shared defaults, and overrides would be in `application-prod.yaml` (or -dev or -test, depending), that seems a bit better to me.
But I can live with either approach.